ex1_bioblend_answers.ipynb

```diff
@@ -339,7 +347,522 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "8) View the results on the Galaxy server with your web browser."
+    "8) Get the id of the output dataset from the workflow. You can get this from the history, or by getting the job id from `gi.workflows.show_invocation()` and then the output dataset id from the job using `gi.jobs.show_job()`. You'll need to wait until the workflow jobs have been scheduled."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'command_version': '',\n",
+      " 'create_time': '2020-07-17T04:25:53.562134',\n",
+      " 'exit_code': 0,\n",
+      " 'galaxy_version': '20.05',\n",
+      " 'history_id': 'e271a7b8bccea52a',\n",
+      " 'id': 'bbd44e69cb8906b5e4d3e25446af7c40',\n",
+      " 'inputs': {'input': {'id': 'bbd44e69cb8906b5d5e5972574c432d6',\n",
+      "                      'src': 'hda',\n",
+      "                      'uuid': 'f0d1b2c7-711f-4c36-8313-2a48e3d2e813'}},\n",
+      " 'model_class': 'Job',\n",
+      " 'outputs': {'out_file1': {'id': 'bbd44e69cb8906b5803d34e5d8b8c823',\n",
+      "                           'src': 'hda',\n",
+      "                           'uuid': '83e8f7b7-5cf8-410d-bee4-55ec067fa24e'}},\n",
+      " 'params': {'__input_ext': '\"txt\"',\n",
+      "            '__workflow_invocation_uuid__': '\"91e340cec7e511eab4a9005056ba55fb\"',\n",
+      "            'chromInfo': '\"/cvmfs/data.galaxyproject.org/managed/len/ucsc/?.len\"',\n",
+      "            'condense': '\"true\"',\n",
+      "            'convert_from': '\"s\"',\n",
+      "            'dbkey': '\"?\"',\n",
+      "            'strip': '\"true\"'},\n",
+      " 'state': 'ok',\n",
+      " 'tool_id': 'Convert characters1',\n",
+      " 'update_time': '2020-07-17T04:28:00.095472'}\n"
+     ]
+    }
+   ],
+   "source": [
+    "invocation = gi.workflows.show_invocation(wf['id'],ret['id'])\n",
```

Comment:
```suggestion
    "invocation = gi.invocations.show_invocation(ret['id'])\n",
```

It would be probably good to replace `ret` with something like `invocation` or `wf_inv` here and above.